### PR TITLE
Rahulyadav guesscount

### PR DESCRIPTION
**Pull request by rahulyadav170923**
bdoin i have made all the necessary changes that we discussed. i have made Drag and Drop components and put most of the game logic in js file. please provide review.

### CODE REVIEW

File: src/activities/guesscount/Guesscount.qml [hunk 1]
```diff
@@ -0,0 +1,529 @@
...
+import "../../core"
+import "guesscount.js" as Activity
+
+ActivityBase {
+    id: activity
+
+    onStart: focus = true
+    onStop: {}
+
+    pageComponent: Rectangle {
```
Comment (petitlapin) [inline src/activities/guesscount/Guesscount.qml]: the background should be the image

File: src/activities/guesscount/Guesscount.qml [hunk 2]
```diff
@@ -0,0 +1,529 @@
...
+            property alias row6: row6
+            property GCAudio audioEffects: activity.audioEffects
+            property var no_of_rows
+            property alias timer: timer
+        }
+
+        onStart: { Activity.start(items) }
+        onStop: { Activity.stop() }
+
+        GCText {
```
Comment (petitlapin) [inline src/activities/guesscount/Guesscount.qml]: it is not used and should be removed

File: src/activities/guesscount/Guesscount.qml [hunk 3]
```diff
@@ -0,0 +1,529 @@
...
+            onHomeClicked: activity.home()
+        }
+
+        Bonus {
+            id: bonus
+            Component.onCompleted: win.connect(Activity.nextLevel)
+        }
+
+        Image {
+            source: Activity.url+"blackboard-2400px.png"
```
Comment (petitlapin) [inline src/activities/guesscount/Guesscount.qml]: the file is missing. It should be the pageComponent

**Comment by petitlapin** [comment]
Hi,

a really quick review:

sometimes, I get: qrc:/gcompris/src/activities/guesscount/guesscount.js:215: TypeError: Cannot read property 'tile' of undefined
qrc:/gcompris/src/activities/guesscount/Guesscount.qml:243: TypeError: Cannot read property 'tile' of undefined
On level 2, passing from sublevel 3 to 4, the two numbers didn't reset to their original place (in "Number" row). The numbers are 10 and 2 and the same for sublevel 3 and 4.

Johnny

**Comment by bdoin** [comment]
You need a README file explaining where comes from background2.jpg. If unsure, just remove it and use a blank background, will make the graphics later. (or pick an existing one or one from http://openclipart.org)

All the other images in svg could be done easily in QML, Just keep the svg button background and write on it.

File: src/activities/guesscount/DragTile.qml [hunk 4]
```diff
@@ -0,0 +1,70 @@
+/* GCompris - guesscount1.qml
```
Comment (bdoin) [inline src/activities/guesscount/DragTile.qml]: Bad file name, should match the actual file

File: src/activities/guesscount/DragTile.qml [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+ *
+ * Copyright (C) 2016 YOUR NAME <xx@yy.org>
```
Comment (bdoin) [inline src/activities/guesscount/DragTile.qml]: Fill the copyright notice.

**Pull request closed without merging**